### ISSUE TITLE
error on clusering app

### ISSUE THREAD

**Issue by Rolandisimo1** [nilanjanchatterjee/Pred_cluster#4]
Error in (function (classes, fdef, mtable) : unable to find an inherited method for function ‘move’ for signature ‘"NULL", "NULL", "POSIXct", "data.frame", "CRS"’
[Uploading data_selectedTime_night_day.csv…]()

**Comment by Alessandro-Araldi-LEC** [comment]
Yes still having the same error if a loaded via CSV or directly from movebank and then transforming from Move2_loc to MoveStack 

Error in (function (classes, fdef, mtable) : unable to find an inherited method for function ‘move’ for signature ‘"NULL", "NULL", "POSIXct", "data.frame", "CRS"’

**Comment by Nova-Scotia** [comment]
Having the same error.
